incubator/xray/templates/xray-analysis-deployment.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "xray-analysis.fullname" . }}
+  labels:
+    app: {{ template "xray-analysis.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: {{ .Values.analysis.name }}
```

Comment:
Does it make sense to make component names configurable?

Reply:
What do you think? I use it only for the selector.